docs/guides/deploy-hyperlane-local-agents.mdx

```diff
@@ -0,0 +1,397 @@
+import TerminologyPartial from "/docs/partials/deploy-hyperlane/_terminology.mdx";
+import SetupKeysPartial from "/docs/partials/deploy-hyperlane/_setup-keys.mdx";
+import DeployContractsPartial from "/docs/partials/deploy-hyperlane/_deploy-contracts.mdx";
+import SendTestMessagesPartial from "/docs/partials/deploy-hyperlane/_send-test-messages.mdx";
+import DeployWarpRoutePartial from "/docs/partials/deploy-hyperlane/_deploy-warp-route.mdx";
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Deploy Hyperlane with Local Agents
+
+:::tip
+
+This guide is for advanced users who have some understanding of configuring and deploying Hyperlane.
+
+If not done so already, beginners should get started by [**deploying Hyperlane with Kurtosis agents**](/docs/deploy-hyperlane).
+
+:::
+
+<TerminologyPartial />
+
+## Overview
+
+There are six steps in this guide:
+
+1. [<b>Set up keys</b>](#1-set-up-keys) that you will use to deploy contracts and run validators and relayers.
```

Comment:
I think this list could be partial-ised as well with the one from `docs/deploy-hyperlane.mdx` > `## Overview`. Also I think the bullet numbering has to be updated here

Reply:
Didn't do this because the kurtosis guide merges agent deployment into one step, whereas the local agent guide has separate steps for validator and relayer